velox/exec/tests/WindowTest.cpp

```diff
@@ -17,129 +17,23 @@
 #include "velox/common/file/FileSystems.h"
 #include "velox/exec/PlanNodeStats.h"
 #include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
 #include "velox/exec/tests/utils/PlanBuilder.h"
 #include "velox/exec/tests/utils/TempDirectoryPath.h"
-#include "velox/functions/lib/window/tests/WindowTestBase.h"
 #include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
 
+using namespace facebook::velox;
 using namespace facebook::velox::exec::test;
```

Comment:
put the test class in anonymous namespace inside facebook::velox::exec

Reply:
Updated, thanks. Just to confirm, this is to be moved to an anonymous namespace in `facebook::velox::exec::test` right?

Reply:
No, in facebook::velox::exec

Reply:
Updated, thanks for confirming.